circle.yml

```diff
@@ -411,15 +411,15 @@ jobs:
                     docker run --rm --privileged $BINFMT_IMAGE --install linux/amd64,linux/arm64
 
                     ## see https://docs.docker.com/desktop/multi-arch/
-                    echo "$DOCKERHUB_PASS" | docker login -u "$DOCKERHUB_USERNAME" --password-stdin
+                    docker login -u "$DOCKERHUB_USERNAME" --password "$DOCKERHUB_PASS"
```

Comment:
These changes were already present in #717 so not sure why they're showing up here. I'm just adding this comment as a note.